en/Obsidian Publish/Security and privacy.md

```diff
@@ -0,0 +1,3 @@
+You choose which notes you want to publish to [[Introduction to Obsidian Publish|Obsidian Publish]]. The rest of your notes stay safe in your vault.
```

Comment:
maybe touch one that one can set one or more passwords to protect their entire site (but not certain files or folders)

Reply:
💯  I'll get on it!

Reply:
💯 I'll get on it!